docs/presto-admin-commands.rst

```diff
@@ -198,7 +198,6 @@ configurations:
     node.launcher-log-file=/var/log/presto/launcher.log
     node.server-log-file=/var/log/presto/server.log
     catalog.config-dir=/etc/presto/catalog
-    plugin.dir=/usr/lib/presto/lib/plugin
```

Comment:
you sure you can omit that instead of explicitly defining it .. the default value might not work..

Reply:
Why do you think so?

https://github.com/prestosql/presto/blob/fca01af7715afd531261343b43dd88d3642e56d6/presto-main/src/main/java/io/prestosql/server/PluginManagerConfig.java#L28

https://github.com/airlift/airlift/blob/6d3d6c968d97415320cf44b646dea4de382af9ba/launcher/src/main/scripts/bin/launcher.py#L176-L179

Reply:
Yeah .. sorry.. commented to early..

Reply:
but something must still be wrong since build fails .. I did not look in detail though